pkg/operator/operator.go

```diff
@@ -112,6 +112,14 @@ func New(config *rest.Config, version, namespace, namespaceSelector, configMapNa
 		UpdateFunc: func(_, newObj interface{}) { o.handleEvent(newObj) },
 	})
 
+	o.openshiftConfigCmapInf = cache.NewSharedIndexInformer(
```

Comment:
separate from your changes, we have so many informers now with handlers that require getting objects that we should consider using a cache.Store to reduce api load and make our reconciliation faster.